### ISSUE TITLE
SpringBoot + Liberty Docker build broken due to change in Open Liberty Docker image permissions

### ISSUE THREAD

**Issue by scottkurz** [OpenLiberty/boost#293]
Failures [here](https://travis-ci.com/OpenLiberty/boost/builds/106379293)
like this:
> [INFO] [info] 03cbdd44157c: Pull complete
[INFO] [info] 651160a46ba7: Extracting
[INFO] [info] 651160a46ba7: Extracting
[INFO] [info] 651160a46ba7: Extracting
[INFO] [info] 651160a46ba7: Extracting
[INFO] [info] 651160a46ba7: Extracting
[INFO] [info] 651160a46ba7: Extracting
[INFO] [info] 651160a46ba7: Extracting
[INFO] [info] 651160a46ba7: Extracting
[INFO] [info] 651160a46ba7: Pull complete
[INFO] [info] Digest: sha256:196d77b785976327b380a7c42d947f949940e6befae69258eee8cf0d6cdc1e45
[INFO] [info] Status: Downloaded newer image for open-liberty:springBoot2
[INFO] [info]  ---> 089290e26aa0
[INFO] [info] Step 2/8 : ARG APP_FILE
[INFO] [info]  ---> Running in 411632c1b977
[INFO] [info]  ---> 61f6f2d905bc
[INFO] [info] Removing intermediate container 411632c1b977
[INFO] [info] Step 3/8 : COPY target/${APP_FILE} /staging/${APP_FILE}
[INFO] [info]  ---> b4804301da64
[INFO] [info] Step 4/8 : RUN springBootUtility thin --sourceAppPath=/staging/${APP_FILE} --targetThinAppPath=/staging/thin-${APP_FILE} --targetLibCachePath=/staging/lib.index.cache
[INFO] [info]  ---> Running in 8c79e749ee52
[INFO] [info] Failed to create directory /staging/lib.index.cache
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] BUILD FAILURE
[INFO] [INFO] ------------------------------------------------------------------------

When configuring "docker-build" locally to recreate remember to use non-dflt config to pull latest image:

```
<configuration>
    <dockerizer>liberty</dockerizer>
    <pullNewerImage>true</pullNewerImage>
</configuration>
```

Quoting from updated Open Liberty Docker Hub [doc](https://hub.docker.com/_/open-liberty):

> Image User
> 
> This image runs by default with USER 1001 (non-root), as part of group 0. Please make sure you read below to set the appropriate folder and file permissions.

**Comment by anjumfatima90** [comment]
@naumanna added these fixes. The build and publish might take an additional day.
https://github.com/OpenLiberty/ci.docker/pull/62
https://github.com/WASdev/ci.docker/pull/243

**Comment by naumanna** [comment]
Turns out my first fix didn't work, but this one will:
https://github.com/OpenLiberty/ci.docker/pull/63
https://github.com/WASdev/ci.docker/pull/244
Once we confirm that fix is good, we will push it out to Docker Hub and IBM Cloud.

**Comment by naumanna** [comment]
The updated images are now available on Docker Hub.

**Comment by scottkurz** [comment]
Resolved by #295.
